pom.xml

```diff
@@ -9,13 +9,14 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
+        <java.version>17</java.version>
+        <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
```

Comment:
Could you explain why do we need it, please? You just want to move those properties to the plugin configuration? If that's the case I'm good with the change

Reply:
@DanielChanger exactly. I've added them in order to use `maven-compiler-plugin`